lc0/build.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
 rm -fr build
-CC=clang CXX=clang++ meson build --buildtype release # -Db_ndebug=true
-# CC=clang CXX=clang++ meson build --buildtype debugoptimized -Db_asneeded=false
-# CC=clang CXX=clang++ meson build --buildtype debug
-cp testdata/* build
-cd build
-ninja
+mkdir build
+pushd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
```

Comment:
FWIW, on Ubuntu 17.10, I had to add `-DCUDA_HOST_COMPILER=/usr/bin/g++-6` since CUDA doesn't support the default GCC7 (don't know if that can be automated).